### PR TITLE
Add `standard-cnpg` image

### PR DESCRIPTION
**Pull request by ianstanton**
Add `standard-cnpg` image. This image contains all extension dependencies for use with Tembo's Standard Stack.

### CODE REVIEW

File: coredb-pg-base/Dockerfile [hunk 1]
```diff
@@ -60,6 +60,16 @@ RUN set -eux; \
           libsystemd-dev \
           libselinux1-dev \
           libzstd-dev \
+	  # required for kafka_fdw
+	  librdkafka1 \
+   	  # required for pgroonga
+   	  libgroonga0 \
```
Comment (nhudson) [inline coredb-pg-base/Dockerfile:66]: Is it me or is the indention off on a few lines?

**Review by nhudson — APPROVED** [review]

File: coredb-pg-base/Dockerfile (hunk 1 above)
Comment (ianstanton) [reply coredb-pg-base/Dockerfile:66]: I think that comment is making it look funny

**Comment by ianstanton** [comment]
@nhudson is the ml-cnpg failure known? Let me know if we can ignore that.

**Comment by nhudson** [comment]
> @nhudson is the ml-cnpg failure known? Let me know if we can ignore that.

@ChuckHend any idea about this?

**Comment by ChuckHend** [comment]
@nhudson @ianstanton -[ this error?](https://github.com/tembo-io/tembo-images/actions/runs/5955615703/job/16154641309?pr=16) I have not seen it before.

```
Removing intermediate container 3ac356dabffa
 ---> df5545b8cd9e
Step 8/17 : RUN set -eux;   trunk install postgresml --version ${PGML_VERSION};   trunk install pgvector --version ${PGVECTOR_VERSION};   trunk install pg_embedding --version ${PGEMBEDDING_VERSION};   trunk install hstore_plpython3u;   trunk install jsonb_plpython3u;   trunk install ltree_plpython3u;
 ---> Running in c31656207a76
+ trunk install postgresml --version 2.7.1
Using pg_config: /usr/bin/pg_config
thread 'main' panicked at 'error occurred: No such file or directory (os error 2)', /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/pg-trunk-0.8.0/src/main.rs:53:6
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
The command '/bin/sh -c set -eux;   trunk install postgresml --version ${PGML_VERSION};   trunk install pgvector --version ${PGVECTOR_VERSION};   trunk install pg_embedding --version ${PGEMBEDDING_VERSION};   trunk install hstore_plpython3u;   trunk install jsonb_plpython3u;   trunk install ltree_plpython3u;' returned a non-zero code: 101

```

File: coredb-pg-base/Dockerfile [hunk 2]
```diff
@@ -60,6 +60,16 @@ RUN set -eux; \
           libsystemd-dev \
           libselinux1-dev \
           libzstd-dev \
+	  # required for kafka_fdw
+	  librdkafka1 \
```
Comment (ChuckHend) [inline coredb-pg-base/Dockerfile:64]: what is `librdkafka1` ? Is it different than `librdkafka-dev`?

Comment (ChuckHend) [reply coredb-pg-base/Dockerfile:64]: i might be wrong, librdkafka-dev might be what we want though

https://github.com/confluentinc/librdkafka#installing-prebuilt-packages

Comment (ChuckHend) [reply coredb-pg-base/Dockerfile:64]: oh is it from the debian repo? i guess im not really sure what version kafka_fdw specifically needs

Comment (ianstanton) [reply coredb-pg-base/Dockerfile:64]: This is the one we added to our test image to resolve the failure: https://github.com/tembo-io/trunk/blob/d64f505839bedb4c8807d1622d35d51d5772e043/images/trunk-test-tembo/Dockerfile#L35. It looks like `-dev` comes with development headers:
```
❯ sudo apt-cache search librdkafka
golang-github-confluentinc-confluent-kafka-go-dev - Apache Kafka Golang client by Confluent
librdkafka++1 - library implementing the Apache Kafka protocol (C++ bindings)
librdkafka-dev - library implementing the Apache Kafka protocol (development headers)
librdkafka1 - library implementing the Apache Kafka protocol
```

File: coredb-pg-base/Dockerfile (hunk 1 above)
Comment (ianstanton) [reply coredb-pg-base/Dockerfile]: You were right. It looked very off when I pulled it up in an IDE

**Comment by ianstanton** [comment]
> @nhudson @ianstanton -[ this error?](https://github.com/tembo-io/tembo-images/actions/runs/5955615703/job/16154641309?pr=16) I have not seen it before. I would not ignore it.
> 
> ```
> Removing intermediate container 3ac356dabffa
>  ---> df5545b8cd9e
> Step 8/17 : RUN set -eux;   trunk install postgresml --version ${PGML_VERSION};   trunk install pgvector --version ${PGVECTOR_VERSION};   trunk install pg_embedding --version ${PGEMBEDDING_VERSION};   trunk install hstore_plpython3u;   trunk install jsonb_plpython3u;   trunk install ltree_plpython3u;
>  ---> Running in c31656207a76
> + trunk install postgresml --version 2.7.1
> Using pg_config: /usr/bin/pg_config
> thread 'main' panicked at 'error occurred: No such file or directory (os error 2)', /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/pg-trunk-0.8.0/src/main.rs:53:6
> note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
> The command '/bin/sh -c set -eux;   trunk install postgresml --version ${PGML_VERSION};   trunk install pgvector --version ${PGVECTOR_VERSION};   trunk install pg_embedding --version ${PGEMBEDDING_VERSION};   trunk install hstore_plpython3u;   trunk install jsonb_plpython3u;   trunk install ltree_plpython3u;' returned a non-zero code: 101
> ```

I'll look into this issue. The changes here are not related to the ml-cnpg image.

**Comment by sjmiller609** [comment]
I think we should do first a different PR:

Rename coredb-pg-base -> tembo-pg-slim
Delete current coredb-pg-slim

Then adjust this PR:

New directory tembo-pg-with-ext-deps (or better name)?
- FROM tembo-pg-slim

**Comment by ianstanton** [comment]
@sjmiller609 Thoughts on naming the new image `tembo-pg-standard`?

**Review by sjmiller609 — APPROVED** [review]

**Comment by sjmiller609** [comment]
> @sjmiller609 Thoughts on naming the new image `tembo-pg-standard` or `standard-cnpg`?

maybe nice to include something indicating it includes extension dependencies, but I try not to get into the naming game too much if I can avoid it :)

**Comment by ianstanton** [comment]
> > @sjmiller609 Thoughts on naming the new image `tembo-pg-standard` or `standard-cnpg`?
> 
> maybe nice to include something indicating it includes extension dependencies, but I try not to get into the naming game too much if I can avoid it :)

Okay, going to merge for now and we can rename if we think of something better!

**Pull request merged**